util/godfig/src/godfig/mod.rs

```diff
@@ -69,7 +79,7 @@ pub mod test {
         let backend = ConfigFile::new(tempfile.into());
         let godfig : Godfig<Test, ConfigFile> = Godfig::new(backend, vec!["test".to_string()]);
 
-        godfig.try_transaction(|data| async move {
+        godfig.try_transaction(|_data| async move {
```

Comment:
It would be good to exercise a transaction on an existing value and make sure it's passed to the closure.